### PR TITLE
enhancement: add storage deposit to native token minting

### LINKED ISSUE #4892: [Task]: Add storage deposit to Mint Native Token Confirmation

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

This PR includes the preparation of the foundry creation  to deduce and display the required storage deposit

## Changelog

```
- add storage deposit to native token minting
```

## Relevant Issues
Closes #4892 

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by nicole-obrien — DISMISSED** [review]
Storage deposit key is showing as the localisation key on the confirmation page.

**Comment by nicole-obrien** [comment]
Also something seems to have broken the alias id selector and it is showing undefined on the mint nft form. Perhaps we can fix that quickly in this pr.

File: packages/shared/components/popups/MintNativeTokenConfirmationPopup.svelte [hunk 1]
```diff
@@ -22,13 +24,35 @@
...
+        ...(description && { description }),
+        ...(url && { url }),
+        ...(logoUrl && { logoUrl }),
+    }
     $: isTransferring = $selectedAccount.isTransferring
 
+    async function _prepareOutput(): Promise<void> {
+        const outputData = buildFoundryOutputData(Number(totalSupply), Number(circulatingSupply), metadata, aliasId)
+        preparedOutput = await $selectedAccount.buildFoundryOutput(outputData)
+        storageDeposit = Number(preparedOutput.amount)
```
Comment (Tuditi) [inline packages/shared/components/popups/MintNativeTokenConfirmationPopup.svelte:45]: Why do you convert this to a number to then convert it back to a string?

**Review by Tuditi — CHANGES_REQUESTED** [review]
Great work! Tested it out and some minor comments:

- There is a locale missing.
image

- In another PR, I refactored the AliasSelector out, but I didn't add a default value to https://github.com/iotaledger/firefly/blob/83b2ac56ef88027764e697989a50da374b4edf26/packages/shared/components/inputs/AliasInput.svelte#L4-L5
This PR seems the perfect place to initialize it back again :smiling_imp:

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**